minidump-common/src/format.rs

```diff
@@ -1634,3 +1635,213 @@ pub struct DSO_DEBUG_64 {
     /// The address of the "dynamic structure"
     pub dynamic: u64,
 }
+
+/// A variable-length UTF-8-encoded string carried within a minidump file.
+#[derive(Clone)]
+pub struct MINIDUMP_STRING {
+    /// The length of the #Buffer field in bytes, not including the `NUL` terminator.
+    ///
+    /// This field is interpreted as a byte count, not a count of Unicode code points.
+    pub length: u32,
+    /// The string, encoded in UTF-8, and terminated with a `NUL` byte.
+    pub buffer: Vec<u8>,
```

Comment:
The linked code is in a linked crate, however. When you check `format.rs`, then you can see all strings declared there are actually `Vec<u8>`. `minidump.rs` actually reads into a String directly.

See https://github.com/luser/rust-minidump/pull/119#discussion_r543496829, I think that it would make sense to parse into a usable structure right away. However, I'd propose to do that in a separate changeset.